README.md

```diff
@@ -28,6 +29,22 @@ If there was no exporter configured or they haven't exported for a while it get
 
 In order to shed some more light in the dark we build a tool called zdb - Zeebe Debugger. It should help you along the way during incidents and broken systems.
 
+
+## Installation
+
+Substitute `<version>` with the version you want to install. Check [releases](https://github.com/Zelldon/zdb/releases) to see all the available versions.
+```
+$ docker exec -it <container_id> /bin/bash
+$ cd /usr/bin
+$ apt-get update
+$ apt-get install curl
+$ curl -O -L https://github.com/Zelldon/zdb/releases/download/<version>/zdb
+$ mkdir target
+$ curl -o target/zdb-<version>.jar -L https://github.com/Zelldon/zdb/releases/download/<version>/zdb-<version>.jar
+$ chmod +x zdb target/zdb-<version>.jar
```

Comment:
Does `chmod +x` provide execution rights for all? I think `chmod u+x` could be slightly better. Also, I don't think the jar needs to be made executable.

Reply:
Makes sense. Updated it.